samples/core/tutorials/estimators/wide.ipynb

```diff
@@ -0,0 +1,1329 @@
+      "cell_type": "code",
+      "source": [
+        "#@title Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+        "# you may not use this file except in compliance with the License.\n",
+        "# You may obtain a copy of the License at\n",
+        "#\n",
+        "# https://www.apache.org/licenses/LICENSE-2.0\n",
+        "#\n",
+        "# Unless required by applicable law or agreed to in writing, software\n",
+        "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+        "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+        "# See the License for the specific language governing permissions and\n",
+        "# limitations under the License."
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "Zr7KpBhMcYvE",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "# TensorFlow Linear Model Tutorial\n",
+        "\n",
+        "<table class=\"tfo-notebook-buttons\" align=\"left\"><td>\n",
+        "<a target=\"_blank\"  href=\"https://colab.research.google.com/github/tensorflow/models/blob/master/samples/core/tutorials/estimators/wide.ipynb\">\n",
+        "    <img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" /><span>Run in Google Colab</span></a>  \n",
+        "</td><td>\n",
+        "<a target=\"_blank\"  href=\"https://github.com/tensorflow/models/blob/master/samples/core/tutorials/estimators/wide.ipynb\"><img width=32px src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" /><span>View source on GitHub</span></a></td></table>"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "77aETSYDcdoK",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "In this tutorial, we will use the `tf.estimator` API in TensorFlow to solve a\n",
+        "standard benchmark binary classification problem: Given census data about a \n",
+        "person such as age, education, marital status, and occupation (the features),\n",
+        "we will try to predict whether or not the person earns more than 50,000 dollars\n",
+        "a year (the target label). We will train a **logistic regression** model, and given \n",
+        "an individual's information our model will output a number between 0 and 1, which\n",
+        "can be interpreted as the probability that the individual has an annual income of over\n",
+        "50,000 dollars.\n",
+        "\n",
+        "Key Point: As a modeler and developer, think about how this data is used and the potential benefits and harm a model's predictions can cause. A model like this could reinforce societal biases and disparities. Is each  feature relevant to the problem you want to solve or will it introduce bias? For more information, read about [ML fairness](https://developers.google.com/machine-learning/fairness-overview/).\n",
+        "\n",
```

Comment:
Maybe add a little blurb at the end (or somewhere in the first paragraph):
"For more information, see the [Estimator guide](https://www.tensorflow.org/guide/estimators)."